stubs/pycocotools/METADATA.toml

```diff
@@ -1,2 +1,5 @@
 version = "2.0.*"
 upstream_repository = "https://github.com/ppwwyyxx/cocoapi"
+requires = ["numpy"]
```

Comment:
```suggestion
requires = ["numpy>=2.0.0rc1"]
```

Reply:
ah, yes i had some checks failing if i do that, particularly python 3.8 on all platforms could not resolve the versions to install. See my previous commits as i attempted to fix that.

I need some help how to disable older python 3.8 checks if i enable numpy 2.0.0